### PR TITLE
Daemon: generate mgr key

### PR DESCRIPTION
**Pull request by AlessandroLorenzi**
fix cap mgr

### CODE REVIEW

File: ceph-releases/luminous/ubuntu/16.04/daemon/start_mgr.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ function start_mgr {
     check_admin_key
 
     # Create ceph-mgr key
-    ceph "${CLI_OPTS}" auth get-or-create mgr."$MGR_NAME" mon 'allow *' -o "$MGR_KEYRING"
+    ceph "${CLI_OPTS}" auth get-or-create mgr."$MGR_NAME" mon 'allow profile mgr' -o "$MGR_KEYRING"
```
Comment (leseb) [inline ceph-releases/luminous/ubuntu/16.04/daemon/start_mgr.sh]: I think we are missing osd and mds too, see: http://docs.ceph.com/docs/master/mgr/administrator/#manual-setup

Comment (leseb) [inline ceph-releases/luminous/ubuntu/16.04/daemon/start_mgr.sh]: @jcsp can you confirm we have it right? Thanks!

Comment (AlessandroLorenzi) [inline ceph-releases/luminous/ubuntu/16.04/daemon/start_mgr.sh]: hi, i'm adding this too

**Comment by leseb** [comment]
please squash everything into a single commit

**Comment by AlessandroLorenzi** [comment]
is it ok now?

**Comment by leseb** [comment]
@AlessandroLorenzi yes thanks!

**Pull request merged**